lib/persistence.js

```diff
@@ -73,6 +85,10 @@ function Persistence (options) {
         });
       }
     });
+
+    self.loadDatabase(function () {
+      self.emit('external-modification');
+    });
   } else {
     self.loadDatabase(function() {});
```

Comment:
The same applies here. It was add here by you some time ago, but I'm not sure what is the reason for it. I think the loadDatabase call should be removed.